packages/reqresp/src/messages/BeaconBlocksByRange.ts

```diff
@@ -15,5 +16,16 @@ export const BeaconBlocksByRange: ProtocolDefinitionGenerator<
     responseType: (forkName) => ssz[forkName].SignedBeaconBlock,
     renderRequestBody: (req) => `${req.startSlot},${req.step},${req.count}`,
     contextBytes: {type: ContextBytesType.Empty},
+    inboundRateLimits: {
+      /**
+       * Nodes uses these endpoint during the sync to fetch blocks.
+       * If we restrict too much we can get into a situation where nodes can't sync from us.
+       * Higher range may end-up in a DOS attack.
+       * So we try to use optimistic values. We can always tune this later.
+       */
+      byPeer: {quota: 500, quotaTime: minutes(1)},
+      total: {quota: 2000, quotaTime: minutes(1)},
+      getRequestCount: (req) => req.count,
```

Comment:
De-duplicate from v2, for example create a beaconBlocksByRangeInboundRateLimits, export from here and import in v2